hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1385,6 +1387,46 @@ def minimize_nodes(self, nodes):
         else:
             raise NotImplementedError
 
+    @defines_shrink_pass()
+    def zero_examples(self, chooser):
+        i = chooser.choose(range(len(self.examples)))
+
+        def trivial_value(ir_type):
+            # TODO: In an ideal world this would take into account the
+            # node kwargs, but this is an adequate first approximation.
+            return {
```

Comment:
I have work on incorporating kwargs into this computation as part of the full IR migration, so this TODO will be addressed in passing in the near future 👍